taxi/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from taxi.views import index
```

Comment:
According to the checklist, there should be a blank line between different groups of imports. Consider adding a blank line after the import statement on line 1.